<launch>
  <arg name="ibeo" default="false"/>
  <arg name="velodyne" default="false"/>
  <arg name="log_level" default="error"/>
  
  <node pkg="nodelet" type="nodelet" name="lidarsegm_pjfa_manager" args="manager" output="screen"/>
  <node if="$(arg ibeo)" pkg="nodelet" type="nodelet" name="ibeo_segmenter" 
      args="load lidarsegm_pjfa/IbeoSegmenter lidarsegm_pjfa_manager">  
    <param name="input_topic" value="/driving/ibeo/spin"/>
    <param  name="output_topic" value="/driving/ibeo/cloud"/>
  </node>

  <node if="$(arg velodyne)" pkg="nodelet" type="nodelet" name="velodyne_segmenter"
      args="load lidarsegm_pjfa/VelodyneSegmenter lidarsegm_pjfa_manager">
      <param name="input_topic" value="/driving/velodyne/processed_pointcloud"/>
      <param name="log_level" value="$(arg log_level)"/>
  </node>
</launch>
